halpybot/packages/case/caseutils.py

```diff
@@ -0,0 +1,104 @@
+"""
+caseutils.py - Crunching the Numbers for Case Information
+
+Copyright (c) The Hull Seals,
+All rights reserved.
+
+Licensed under the GNU General Public License
+See license.md
+"""
+from __future__ import annotations
+import re
+from typing import Dict, TYPE_CHECKING
+from pendulum import now
+from attrs import evolve
+from ..models import Case, Platform
+
+if TYPE_CHECKING:
+    from ..ircclient import HalpyBOT
+
+
+async def format_case_details(case: Case) -> str:
```

Comment:
its functions like these that make me wonder whether jinja templates are better than this giant pile of string manipulation

Reply:
Unfamiliar with jinja templates. Will discuss in side channel.